heroic-core/src/test/java/com/spotify/heroic/cluster/CoreClusterManagerTest.java

```diff
@@ -55,7 +56,7 @@ public void setUp() {
         final boolean useLocal = true;
 
         manager = spy(new CoreClusterManager(async, discovery, localMetadata, protocols, scheduler,
-            useLocal, options, local, context, ImmutableSet.of()));
+            useLocal, options, local, context, ImmutableSet.of(), NoopQueryReporter.get()));
```

Comment:
Mock `QueryReporter`, and possibly add some verifications to existing tests (or new tests)?